client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java

```diff
@@ -113,6 +113,8 @@
 import java.util.Map;
 import java.util.StringJoiner;
 
+import static org.opensearch.internal.client.Requests.INDEX_CONTENT_TYPE;
```

Comment:
We have to figure out how to completely get rid of dependency on `org.opensearch.internal.client.Requests`

Reply:
For rest-high-level client the only dependency from internal client `Requests` is  `INDEX_CONTENT_TYPE` (which is `MediaTypeRegistry.JSON` coming from `:server` `org.opensearch.core.xcontent`).